src/gui/qgslistwidget.h

```diff
@@ -0,0 +1,92 @@
+/***************************************************************************
+    qgslistwidget.h
+     --------------------------------------
+    Date                 : 08.2016
+    Copyright            : (C) 2016 Patrick Valsecchi
+    Email                : patrick.valsecchi@camptocamp.com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSLISTWIDGET_H
+#define QGSLISTWIDGET_H
+
+#include "qgstablewidgetbase.h"
+#include <QAbstractTableModel>
+#include <QVariant>
+
+///@cond PRIVATE
```

Comment:
Is this actually private or just not available in python bindings? Seems to me it should be public but with the unavailable note.